resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
We'll need to track from which version of k8s and associated ocp is this valid

Reply:
The v1 update was include in 1.16 of kubernetes which is 4.3 of OCP I believe

Reply:
can you please add this to documentation? its very important that we point out that if they're using older k8s version then they can use v0.0.2 of ripsaw